### ISSUE TITLE
A test issue for keyboard-only navigation

### ISSUE THREAD

**Issue by mikethemelanson** [mikethemelanson/tester-for-actions#1]
What did I notice?  Well, first, let me try to add some styling from the toolbar using my keyboard:

> This should be an indented quote style. 

Okay, that inserted the markdown symbol. Took a little bit of figuring out. I can seem to select text and then apply a shortcut to it. Let's try with a link - nope, can't select and then navigate away. 

```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```
Okay, I can do various things, definitely not without difficulty.
